core/ir/riscv64/isl/xtheadcmo.txt

```diff
@@ -0,0 +1,14 @@
+# XTheadCmo (Cache Management Operations)
+# Version 1.0
+# https://github.com/T-head-Semi/thead-extension-spec
+
+# Only list instructions that are valid in usermode.
```

Comment:
Note that we would like to cover privileged as well, to enable various uses like the standalone drdecode library for decoding arbitrary bits, or e.g. with hardware kernel tracing (currently only on x86) we end up stitching kernel instructions into drmemtrace traces.  How about a TODO comment.

Reply:
> with hardware kernel tracing

This is cool! Added TODOs.